3rd-party/class.jetpack-amp-support.php

```diff
@@ -49,6 +52,28 @@ public static function init() {
 
 		// Sync the amp-options.
 		add_filter( 'jetpack_options_whitelist', array( 'Jetpack_AMP_Support', 'filter_jetpack_options_whitelist' ) );
+
+		// Show admin bar.
+		add_filter( 'show_admin_bar', array( 'Jetpack_AMP_Support', 'show_admin_bar' ) );
```

Comment:
I do not think this is right. It is causing the admin bar to be hidden on all AMP pages, even when it should be displayed. 

In AMP plugin 1.3 we introduced AMP dev mode support to exclude the admin bar from being subject to AMP validation constraints, allowing administrative scripts and styles to work as expected even on AMP pages. 

More info: https://weston.ruter.net/2019/09/24/integrating-with-amp-dev-mode-in-wordpress/

Reply:
@jeherve if the issue is the scripts and styles that the Jetpack Likes module includes, a better way to resolve this I think is to explicitly mark them for AMP dev mode by adding `admin-bar` as a dependency for the scripts and styles.

Reply:
See also https://github.com/ampproject/amp-wp/pull/4001

Reply:
Thanks for jumping in. We'll need to revisit this. I've logged your remarks in #15353.